src/app/player/player.component.ts

```diff
@@ -38,7 +39,10 @@ export class PlayerComponent implements OnChanges, OnInit {
   }
 
   ngOnInit() {
-    this.player = new DovetailPlayer(this.audioUrl);
+    this.player = new DovetailAudio(this.audioUrl);
```

Comment:
Most of the rest of the changes in this file are inconsequential, except for the adstart and adend listeners